### PR TITLE
Well events

### PR DESCRIPTION
**Pull request by joakim-hove**
@atgeirr ??

### CODE REVIEW

File: opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp [hunk 1]
```diff
@@ -467,7 +471,7 @@ namespace Opm {
                 updateWellStatus( *well , currentStep , status );
                 if (well->setProductionProperties(currentStep, properties))
                     m_events.addEvent( ScheduleEvents::PRODUCTION_UPDATE , currentStep);
```
Comment (atgeirr) [inline opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp]: Should not this (and injection, polymer) update also insert a well production update event?

Comment (joakim-hove) [inline opm/parser/eclipse/EclipseState/Schedule/Schedule.cpp]: Yes - update coming.

**Comment by joakim-hove** [comment]
The Polymer event might be a little too detailed?

**Comment by atgeirr** [comment]
> The Polymer event might be a little too detailed?

Perhaps, I just looked for some kind of symmetry between field and well events.

**Comment by joakim-hove** [comment]
OK - I assume this is *Good Enough*&trade; - we can improve it later if necessary.

**Pull request merged**